datafusion/sqllogictest/test_files/array_aggregate.slt

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#############
+## Array Aggregate Tests
+#############
+
+# sum
+
+query IRIIIRII
+select 
+array_aggregate([1, 3, 5, 7], 'sum'),
```

Comment:
```suggestion
array_aggregate([], 'sum'),
array_aggregate([null], 'sum'),
select array_aggregate([1, 3, 5, 7], 'SUM'),
array_aggregate([1, 3, 5, 7], 'sum'),
```

Reply:
I intend to skip this since null is not yet handle correctly